### ISSUE TITLE
Build Connections page with ORCID and SSO Connections

### ISSUE THREAD

**Issue by bofstein** [CDLUC3/dmsp_frontend_prototype#128]
Build the page seen here, but without Github and ResearchGate for now: https://www.figma.com/proto/RRtoRLyr5Spf0KiyRfEclt/DMP-Explain?node-id=771-2357&t=LIKQ4JuadoMKIEAS-8&scaling=scale-down-width&content-scaling=fixed&page-id=736%3A1383&starting-point-node-id=736%3A1772&show-proto-sidebar=1&hotspot-hints=0&hide-ui=1

Decide on having icon in button for ORCID or making it to the header of the section

Just FE for now, won't be functional yet

**Comment by bofstein** [comment]
Make sure to follow https://info.orcid.org/brand-guidelines/

**Comment by bofstein** [comment]
Also need to consider how it looks if ORCID or SSO is already connected.  Both should show what the connection is (e.g. ORCID ID or SSO connection), and have a button to Disconnect

**Comment by briri** [comment]
Screenshot of current disconnect:
![Image](https://github.com/user-attachments/assets/7130ce99-67e6-4776-a0fc-6b6bf4e7beb1)

**Comment by bofstein** [comment]
Acceptance Criteria:
- Page exists on Dev page
- Page has sections for ORCID and SSO connections
- Each section has a description of the connection and a button to connect (if not already connected)
- ORCID logo usage follows their brand guidelines

@jupiter007 We said in this ticket it didn't have to be functional yet, but then we talked in meetings about getting you the proper credentials to hook up. Let me know if you want or can do any of these in this ticket or would want them in a separate ticket:

- If a user is connected to either, it displays the connection information
- User has the ability to click a button to remove the connection
- This removal has some sort of double confirmation
- Clicking the Connect button brings up a new window to log into the external site to form the connection

**Comment by jupiter007** [comment]
Just a note that I sent a message to ORCID to confirm that it is ok to use their ORCID iD icon in the button, as is proposed in the design, because based on reading the guidelines, it wasn't clear whether that would be allowed.

**Comment by jupiter007** [comment]
Even though we don't have the database table or API endpoint created to add the access token from ORCiD, I added the functionality to the code using localStorage for now. I also created the backend ticket to create the db table for external api access tokens: https://github.com/CDLUC3/dmsp_backend_prototype/issues/116 and another ticket for the frontend to make the updates once the API is in place: https://github.com/CDLUC3/dmsp_frontend_prototype/issues/134

**Comment by jupiter007** [comment]
We were only able to set up a redirect_uri to our dev server, (https://ui.dmphub.uc3dev.cdlib.net/users/auth/orcid/callback), since ORCiD only allows "https" redirect urls. So we can't test that it's working until it gets pushed up to the dev server after merging to development branch.

However, you can test what the interaction will look like by clicking on the button under the "Test orcid" section of the page.
